.travis.hhvm.ini

```diff
@@ -0,0 +1,2 @@
+hhvm.enable_zend_compat = true
```

Comment:
I would rather disable the HHVM job

Reply:
I think there is not a HHVM job now! Should I delete this file? I don't remember exactly why I added this option, perhaps to pass a Travis job. Please let me know what's the best option.